docs/manual/contributors.tex

```diff
@@ -97,4 +98,5 @@
 Vlastimil Dort,
 Weitian Xing,
 Werner Dietl,
-Zhiping Cai.
+Zhiping Cai,
+nimakarimipour.
```

Comment:
Is that a different person from `Nima Karimipour` above?

Reply:
Fixed, good catch!

Reply:
This needed to be fixed in the plume-scripts repository, or the problem will recur.  I have [fixed it](https://github.com/plume-lib/plume-scripts/commit/22c571c671190836c020fdf8e7cbab5c0803e0a6).